packages/shopping/src/application.ts

```diff
@@ -160,12 +147,12 @@ export class ShoppingApplication extends BootMixin(
       if (file.endsWith('.yml')) {
         const userFile = path.join(usersDir, file);
         const yamlString = YAML.parse(fs.readFileSync(userFile, 'utf8'));
-        const input = new NewUser(yamlString);
-        const password = await passwordHasher.hashPassword(input.password);
-        input.password = password;
-        const user = await userRepo.create(_.omit(input, 'password'));
-
-        await userRepo.userCredentials(user.id).create({password});
+        const userWithPassword = new UserWithPassword(yamlString);
+        const userManagementService = new UserManagementService(
+          userRepo,
+          passwordHasher,
+        );
```

Comment:
Let's use `this.get(UserServiceBindings.USER_SERVICE)`. This will leverage dependency injection to set up `UserManagementService`.

Reply:
Updated accordingly